src/display/views/help/components/ContactFormPane.js

```diff
@@ -1,26 +1,17 @@
 import React from 'react';
-import {
-	Row,
-	Col,
-	TabPane,
-	Button,
-	FormGroup,
-	Input,
-	Label,
-	CardHeader,
-	CardBody
-} from 'reactstrap';
+import { TabPane, CardHeader, CardBody } from 'reactstrap';
 import { AvForm } from 'availity-reactstrap-validation';
-import PropTypes from 'prop-types';
+// import PropTypes from 'prop-types';
 import Card from '../../../shared/styled/Card';
 
-const propTypes = {
-	submitContactForm: PropTypes.func.isRequired
-};
+// const propTypes = {
```

Comment:
Feel free to remove unused code rather than just commenting it out. It's much cleaner; the nice thing about version control is we can always go back and find this code if we need it again for some reason. :)

Reply:
Learned about version control very recently. Forgot about it while working on it (my bad). Thank you for making me remember and I am on it. Thank you for pointing it out!